app/plot/plotter.py

```diff
@@ -415,7 +421,11 @@ def create_filters(self):
             "header_background": self.config["background_fill_color"],
         }
         self.filters["multi_choice"] = Accordion(
-            ("Branchen auswählen", filters_multi_choice), **filter_options, active=[0]
+            ("Branchen auswählen", filters_multi_choice),
+            header_color="#1E3791",
+            active_header_background="#F6F6F6",
+            header_background="#F6F6F6",
+            active=[0],
```

Comment:
It's maybe because it's a draft, but are you sure you want to change how you invoke the Class rather than the filter_options dict?

Reply:
It's a little misleading but this active attribute does not really correspond to the filters itself but only makes sure, that the 0-position of the wrapper accordion is open, ergo showing the legend

Reply:
fixed